x-pack/plugins/security_solution/public/overview/components/entity_analytics/anomalies/columns.tsx

```diff
@@ -52,13 +50,15 @@ export const useAnomaliesColumns = (loading: boolean): AnomaliesColumns => {
       },
       {
         field: 'count',
-        sortable: ({ count, status }) => {
+        sortable: ({ count, job }) => {
           if (count > 0) {
             return count;
           }
-          if (status === AnomalyJobStatus.disabled) {
+
+          if (job && isJobStarted(job.jobState, job.datafeedState)) {
             return -1;
           }
+
           return -2;
```

Comment:
unrelated to your changes but do you know how are these negative numbers used?

Reply:
Yes, it overrides the column numeric sort order. When the job is uninstalled, we set a negative number to display it at the bottom of the table. So we have the following sort order: 
Jobs with anomalies
Instaled jobs without anomalies
Uninstalled jobs

I can simplify it a bit by using `0` and `-1` instead of `-1` and `-2`